forms/mainwindow.ui

```diff
@@ -233,23 +233,23 @@
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
```

Comment:
Please revert all changes to this file.

Reply:
I hate how Qt always insists on dicking around with this file when you don't mean to... and also explicitly prevents you from making changes in the text view.